### ISSUE TITLE
Examples of Violin plot are using older api

### ISSUE THREAD

**Issue by harshnisar** [mwaskom/seaborn#579]
I was trying examples from http://stanford.edu/~mwaskom/software/seaborn-dev/generated/seaborn.violinplot.html when I stumbled on an error and later found out on SO that I was using the older API of the violin plot method. 

Even this in my understanding is an example with an old api, http://stanford.edu/~mwaskom/software/seaborn-dev/generated/seaborn.stripplot.html

**Comment by mwaskom** [comment]
Which example in particular in the `stripplot` API docs do you mean? That function actually didn't exist in 0.5.1. The generated API docs should be up to date (actually this is enforced by doctests).

**Comment by harshnisar** [comment]
So I tried searching for the page where I found the example of violin plot with an older api, but couldn't. 
Then I googled *violin plot seaborn* and the first link that was shown was this: http://stanford.edu/~mwaskom/software/seaborn/generated/seaborn.violinplot.html

This is the documentation for 0.5.1 version and the one which was causing the confusion. I am using version 0.6.

How can this confusion be resolved? Which one is considered stable. Should there be some marker on the documentation?
